### DISCUSSION TITLE
Auto Discovery on different subnet?

### DISCUSSION THREAD

**Discussion by EricSabe** [plasticrake/homebridge-tplink-smarthome#243] [General]
All of my TP-Link gear is subnet B. Homebridge is running in Docker on Synology on subnet A.  PfSense is the firewall/router.  I've been using static DHCP assignment and Manual Discovery to surface new accessories.  Is there another way?  I haven't found an "official" pfSense package for a UDP broadcast relay.  Avahi seems to only support mDNS.

**Comment by plasticrake** [comment]
I don't use it, but have you seen this? https://github.com/marjohn56/udpbroadcastrelay

**Reply by EricSabe** [reply]
Hrmmm... looks simple enough.  I'll give it a go.

**Comment by EricSabe** [comment]
Update: Works like a charm.  Requires a FW rule that allows response traffic from the IOT subnet to the Homebridge subnet.  UDP/SRC: 9999/DST: [random port above 50k??].

**Reply by EricSabe** [reply]
Oh, I see you've made it possible to lock the destination port in settings:
![image](https://user-images.githubusercontent.com/91292218/153057377-6f76fa9d-aa7e-45a7-9e45-ee7da462b931.png)

**Comment by plasticrake** [comment]
Yeah that is the port that is opened up on the homebridge machine for the discovery packets
